docs/source/updating_capabilities.md

```diff
@@ -0,0 +1,341 @@
+topic for many of the commands.
+
+As with any channel configuration update, updating capabilities is, at a high
+level, a three step process (for each channel):
+
+1. Get the latest channel config
+2. Create a modified channel config
+3. Create a config update transaction
+
+We will enable these capabilities in the following order:
+
+1. [Orderer system channel](#orderer-system-channel-capabilities)
+
+  * Orderer group
+  * Channel group
+
+2. [Application channels](#enable-capabilities-on-existing-channels)
+
+  * Orderer group
+  * Channel group
+  * Application group
+
+While it is possible to edit multiple parts of the configuration of a channel
+at the same time, in this tutorial we will show how this process is done incrementally.
+In other words, we will not bundle a change to the `Orderer` group and the `Channel`
+group of the system channel into one configuration change. This is because not
+every release will have both a new `Orderer` group capability and a `Channel`
+group capability.
+
+Note that in production networks, it will not be possible or desirable for one
+user to be able to update all of these channels (and parts of configurations)
+unilaterally. The orderer system channel, for example, is administered exclusively
+by ordering organization admins (though it is possible to add peer organizations
+as ordering service organizations). Similarly, updating either the `Orderer` or
+`Channel` groups of a channel configuration requires the signature of an ordering
+service organization in addition to peer organizations. Distributed systems
+require collaborative management.
+
+#### Create a capabilities config file
+
+Note that this tutorial presumes that a file called `capabilities.json` has been
+created and includes the capability updates you want to make to the various
+sections of the config. It also uses `jq` to apply the edits to the modified
+config file.
+
+Note that you are not obligated to create a file like `capabilities.json` or to
+use a tool like `jq`. The modified config can also be edited manually (after it
+has been pulled, translated, and scoped). Check out this
+[sample channel configuration](./config_update.html#sample-channel-configuration)
+for reference.
+
+However, the process described here (using a JSON file and a tool like `jq`) does
+have the advantage of being scriptable, making it suitable for proposing
+configuration updates to a large number of channels. This is why it is **the
+recommended way to update channels**.
+
+In this example, the ``capabilities.json`` file looks like this (note: if you
+are updating your channel as part of [Upgrading to the latest version of Fabric](./upgrade_to_newest_version.html)
+you will need to set the capabilities to the levels appropriate to that release):
+
+```
+   {
+     "channel": {
+         "mod_policy": "Admins",
+             "value": {
+                 "capabilities": {
+                     "V1_4_3": {}
```

Comment:
these should all be udpated to V2_0 now right?

Reply:
No, this is just a sample code snippet. It could by anything --- I don't plan to change this with each release.